src/Runner.Worker/StepsRunner.cs

```diff
@@ -83,171 +82,170 @@ public async Task RunAsync(IExecutionContext jobContext)
                 step.ExecutionContext.ExpressionFunctions.Add(new FunctionInfo<SuccessFunction>(PipelineTemplateConstants.Success, 0, 0));
                 step.ExecutionContext.ExpressionFunctions.Add(new FunctionInfo<HashFilesFunction>(PipelineTemplateConstants.HashFiles, 1, byte.MaxValue));
 
-                // Initialize scope
-                if (InitializeScope(step, scopeInputs))
-                {
-                    // Populate env context for each step
-                    Trace.Info("Initialize Env context for step");
+                step.ExecutionContext.ExpressionValues["steps"] = step.ExecutionContext.StepsContext.GetScope(step.ExecutionContext.ScopeName);
```

Comment:
What's different about this file?

`if (InitializeScope[...]` was removed and the different indent-level is causing too much red/green. I can't tell what changed.

I see the InitializeScope method was deleted. Anything else?

Reply:
@ericsciple  InitializeScope was completely removed.

Also, most of the CompleteStep was removed so that it looks like. All of the code removed related to the "Scopes" attribute of the ExecutionContext which is no longer used afaik:
```
private void CompleteStep(IStep step, TaskResult? result = null, string resultCode = null)
{
  var executionContext = step.ExecutionContext;
  executionContext.Complete(result, resultCode: resultCode);
}
``

see: https://github.com/actions/runner/pull/568/files#diff-4fa5184d693636c90a417707cf39d422L468-L522